adminsortable/templates/adminsortable/csrf/jquery.django-csrf.html

```diff
@@ -16,7 +16,7 @@
         return cookieValue;
     }
 
-    var csrftoken = getCookie('{{ csrf_cookie_name }}');
+    var csrftoken = '{{ csrf_token }}' || getCookie('{{ csrf_cookie_name }}');
```

Comment:
@alsoicode I have concluded that this is the only needed code to make the CSRF work with session based CSRF. Technically the `getCookie` seems redundant now as we will always have the `csrf_token` in the context because of the CSRF middleware. 

What would you suggest here?

Reply:
If the `csrf_token` value is already present, then yes, let's just use that, otherwise, fallback to the cookie. I'm going to close this PR and add this change to others that I've made for Django 2 compatibility so I can get a new release out with everything.